app/controllers/puppetclasses_controller.rb

```diff
@@ -85,4 +86,8 @@ def action_permission
         super
     end
   end
+
+  def fetch_associations
+    @puppetclass.lookup_keys.includes(:lookup_values)
```

Comment:
I think this needs to happen *before* the .find, probably by overriding `resource_scope`?

    def resource_scope
      super.includes(:lookup_keys => [:lookup_values])
    end

At the moment it's preloading it here, but it's not having an effect later when the view calls `@puppetclass.lookup_keys[0].lookup_values` or something, so causes a second query (and Bullet warning).

You can replicate this in the console, if you compare:

    pc = Puppetclass.includes(:lookup_keys => [:lookup_values]).find('example')
    pc.lookup_keys.first.lookup_values

to

    pc = Puppetclass.find('example')
    pc.lookup_keys.includes(:lookup_values)
    pc.lookup_keys.first.lookup_values

The latter will generate extra queries.